openmetadata-ui/src/main/resources/ui/src/styles/components/table.less

```diff
@@ -95,4 +95,7 @@
       background: @active-color;
     }
   }
+  .expand-cell-empty-icon-container {
+    display: none !important;
+  }
```

Comment:
Why we need this?

Reply:
If collapse icon is not visible for rows not having children, it takes a width.
So we need this.